src/early_multicellular_stage/editor/EarlyMulticellularEditor.cs

```diff
@@ -555,16 +555,26 @@ private void UndoEndCellTypeEditAction(EndCellTypeEditActionData data)
     [DeserializedCallbackAllowed]
     private void DoStartCellTypeEditAction(StartCellTypeEditActionData data)
     {
-        if (selectedCellTypeToEdit == data.StartedCellTypeEdit)
-            return;
-
-        // This fixes complex cases where multiple types are redoing actions
-        selectedCellTypeToEdit = data.StartedCellTypeEdit;
-        cellEditorTab.OnEditorSpeciesSetup(EditedBaseSpecies);
+        SwapEditingCell(data.StartedCellTypeEdit);
     }
 
     [DeserializedCallbackAllowed]
     private void UndoStartCellTypeEditAction(StartCellTypeEditActionData data)
     {
+        SwapEditingCell(data.StartedCellTypeEdit);
```

Comment:
Hang on, help me understand what's going on here... So the way this works is that when a cell start action is undone, it still swaps to that cell type edit? If I remember right the undo history should be something like:
```
start cell type|organelle edit|end cell type|start different cell type|organelle edit
```
and going back in the history for a start cell type being undone should already work. Well I guess it doesn't really work if this change fixes the crash (I originally thought this was not necessary). The clarification on why this is necessary hit to performance (needing to start a cell type edit just to overwrite it immediately afterwards) should then be put in the code.